include/wctype.h

```diff
@@ -15,8 +15,9 @@ extern "C" {
 
 typedef _PDCLIB_wint_t wint_t;
 
-// wctrans_t
-// wctype_t
+// Xbox-specific hack to make this header work
```

Comment:
nit: Use "types" instead of "hacks" so people know the scope of this comment.

Also, the prefix is missing from commit title